registration/tests.py

```diff
@@ -7,12 +7,23 @@
 from rest_framework.test import APIClient
 from rest_framework.authtoken.models import Token
 from rest_framework import status
+from requests.adapters import HTTPAdapter
+from requests_testadapter import TestSession, Resp
+from go_http.contacts import ContactsApiClient
+from fake_go_contacts import Request, FakeContactsApi
 from .models import Source, Registration, fire_jembi_post
-from .tasks import jembi_post_json
-from .tasks import Jembi_Post_Json
+from registration import tasks
 
 
-Jembi_Post_Json.get_timestamp = lambda x: "20130819144811"
+def override_get_timestamp():
+    return "20130819144811"
+
+
+def override_get_tomorrow():
+    return "2014-01-02"
+
+tasks.get_timestamp = override_get_timestamp
+tasks.get_tomorrow = override_get_tomorrow
```

Comment:
@imsickofmaps Oh! Haha :laughing: I hear what you're saying, however the different dates are used for calibration in the next issue (#90) - I'm repeating the same tests that we had in JS code to validate that we're getting the same results.  So I'm going to land this here and if you're happy I'm getting the correct results in the next issue I can change it then.